### PR TITLE
fix(esl-event-listener): fix `ESLEventTargetDecorator` dispatched event `target`

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by fshovchko — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

File: src/modules/esl-event-listener/test/targets/decorator.test.ts [hunk 1]
```diff
@@ -24,6 +25,24 @@ describe('ESLEventUtils.decorate proxy', () => {
...
+    const handler = jest.fn();
+
+    beforeEach(() => dec.addEventListener('click', handler));
+    afterEach(() => dec.removeEventListener('click', handler));
+
+    test('ESLEventUtils.decorate does not replace event.target', () => {
+      et.dispatchEvent(new Event('click'));
+      expect(handler).lastCalledWith(expect.objectContaining({target: et}));
+    });
+    test('ESLEventUtils.decorate does not replace event.target', () => {
```
Comment (ala-n) [inline src/modules/esl-event-listener/test/targets/decorator.test.ts:40]:
```suggestion
    test('ESLEventUtils.decorate use ESLEventTargetDecorator instance as `event.currentTarget`', () => {
```

File: src/modules/esl-event-listener/test/targets/decorator.test.ts [hunk 2]
```diff
@@ -24,6 +25,24 @@ describe('ESLEventUtils.decorate proxy', () => {
...
+    const handler = jest.fn();
+
+    beforeEach(() => dec.addEventListener('click', handler));
+    afterEach(() => dec.removeEventListener('click', handler));
+
+    test('ESLEventUtils.decorate does not replace event.target', () => {
+      et.dispatchEvent(new Event('click'));
+      expect(handler).lastCalledWith(expect.objectContaining({target: et}));
+    });
+    test('ESLEventUtils.decorate use ESLEventTargetDecorator instance as `event.currentTarget`', () => {
```
Comment (fshovchko) [inline src/modules/esl-event-listener/test/targets/decorator.test.ts:40]:
```suggestion
    test('ESLEventUtils.decorate uses ESLEventTargetDecorator instance as `event.currentTarget`', () => {
```

**Review by fshovchko — APPROVED** [review]

File: src/modules/esl-event-listener/test/targets/decorator.test.ts [hunk 3]
```diff
@@ -24,6 +25,24 @@ describe('ESLEventUtils.decorate proxy', () => {
     });
   });
 
+  describe('ESLEventUtils.decorate event target',  () => {
+    const et = document.createElement('div');
```
Comment (dshovchko) [inline src/modules/esl-event-listener/test/targets/decorator.test.ts:29]: typo: `el`

**Review by dshovchko — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Pull request merged**